torch/csrc/jit/ir/ir.cpp

```diff
@@ -1595,6 +1595,21 @@ Node* Graph::createTupleSlice(Value* tup, int64_t beg, int64_t end) {
   return n;
 }
 
+Node* Graph::createEnumName(Value* e) {
+  assert(e->type()->cast<EnumType>());
+  auto n = create(prim::EnumName, {e});
+  n->output()->setType(StringType::get());
+  return n;
+}
+
+Node* Graph::createEnumValue(Value* e, const TypePtr& enum_val_type) {
+  auto enum_type = e->type()->cast<EnumType>();
+  assert(enum_type);
```

Comment:
FYI, we have two types of assert's we use:   `TORCH_CHECK`, and `TORCH_INTERNAL_ASSERT`. The first is for user errors and the second is for developer invariants. In this case,  it would be a developer invariant. But you don't need to use either here because we have `e->type()->expect<EnumType>();` which will throw if it's not the right type.

Reply:
Done, replaced with expect()